x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/data_sources/monitoring_entity_source_data_client.test.ts

```diff
@@ -49,8 +51,8 @@ describe('MonitoringEntitySourceDataClient', () => {
     dataClient = new MonitoringEntitySourceDataClient(defaultOpts);
   });
 
-  describe('init', () => {
```

Comment:
this was copy pasta. the create method was copied from the init method of the engine descriptor client